packages/p2p/src/components/online-status/online-status-label.jsx

```diff
@@ -11,35 +11,52 @@ const OnlineStatusLabel = ({ is_online, last_online_time, size = isMobile() ? 'x
             if (last_online_time) {
                 const start_time = epochToMoment(last_online_time).unix();
                 const end_time = toMoment().unix();
-
                 const diff = getDiffDuration(start_time, end_time);
-                const addPlural = duration => (duration !== 1 ? 's' : '');
 
                 if (diff.years()) return localize('Seen more than 6 months ago');
                 if (diff.months()) {
                     if (diff.months() > 6) {
                         return localize('Seen more than 6 months ago');
                     }
-                    return localize('Seen {{ duration }} month{{ prural }} ago', {
+                    if (diff.months === 1) {
+                        return localize('Seen {{ duration }} month ago', {
+                            duration: diff.months(),
+                        });
+                    }
+                    return localize('Seen {{ duration }} months ago', {
```

Comment:
I'm not 100% sure if these approach would work:
```
localize('Seen {{ duration }} {{ unit }} ago`, {
  duration: diff.months(),
  unit: diff.months() === 1 ? 'month' : 'months'
})
```
or using nested localize:
```
localize('Seen {{ duration }} {{ unit }} ago`, {
  duration: diff.months(),
  unit: diff.months() === 1 ? localize('month') : localize('months')
})
```
but just to be safe, since at the moment translations for these have not been added yet and I can't test if these would work, I made the logic for singular and plural strings to be in separate conditions.